### DISCUSSION TITLE
How write Brush Strokes  in tex

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#701] [Q&A] [answered]
like this img

![未命名-1](https://user-images.githubusercontent.com/17884664/149807086-1b397f20-9b8f-445e-a27d-b511ec7fa2ed.jpg)

**Comment by Danielku15** [comment, marked as answer]
This is currently not possible. The open issue related to this is: https://github.com/CoderLine/alphaTab/issues/505 You added a comment there already in the past.
